### DISCUSSION TITLE
Dimming the screen further then the monitor defaults

### DISCUSSION THREAD

**Discussion by RLFontan** [xanderfrangos/twinkle-tray#160] [Q&A]
Hey guys, how it's going on?

I'm sorry if this is an inappropriate place, but I would like to ask a question about the practice of dimming the screen further than monitor defaults related to the Twinkle Tray app.

One thing I always liked about Linux, is how I could use Xrandr or Xrandr based applications to dimmer my screen beyond monitor defaults. The first time I found an application on Windows Store that could do this, was Twinkle Tray. That was a long time ago, and then when I was using Windows, I realized that some Twinkle Tray updates took my ability to do that, and I could only dim my screen as much as my physical buttons would allow.

**TL;DR. My question is: there was a specific technical reason why Twinkle Tray couldn't dim the monitor brightness more than physical buttons default anymore? For example, the developers were afraid that it could harm some monitors?**

Thank you for your attention. 

PS: If this isn't the appropriate place to make these kinds of questions, I'm sorry.

**Comment by xanderfrangos** [comment]
@RLFontan Here is good. 

There are two additional things that Twinkle Tray _could_ do to further dim a display:

**1. Contrast control -** Adjusting contrast in addition to brightness can greatly affect how bright/dim a display is. The side effect is that the colors are less accurate, but most people don't care too much about color accuracy. A future update will allow contrast to be adjusted (where possible).
**2. Black overlay -** Some apps fake brightness adjustments by putting a full-screen, semi-transparent black window over everything. Because it doesn't really change the brightness, it doesn't actually send any less (or more) light to your eye. It's just a darker color. This works OK for some displays, but is never as good as changing the real backlight brightness. I don't intend to ever add this functionality to Twinkle Tray.
